package.json

```
@@ -16,7 +16,8 @@
     "test:lint": "eslint --ext ts src",
     "update": "yarn",
     "build": "tsc -p .",
-    "watch": "tsc -p . -w"
+    "watch": "tsc -p . -w",
+    "develop": "yarn build && yarn start"
```

`dev` is a better name:
```suggestion
    "dev": "yarn build && yarn start"
```